### PR TITLE
CMS editor and page layout

### PR DESCRIPTION
**Pull request by hoptond**
Merges the CMS editor pages into master. Mostly, I want to know if the links go to where they should and the pages display correctly. Any problems in the HTML would be nice, too. There isn't any functionality (yet).

### CODE REVIEW

**Review by Benson-J — COMMENTED** [review]
All good, code is correct & links go to the right places.

File: edit/about.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<?php
+/**
```
Comment (mporam) [inline edit/about.php]: remove this comment

File: edit/about.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+ * Created by PhpStorm.
+ * User: academy
+ * Date: 01/10/2018
+ * Time: 09:39
+ */
+
+
+
+?>
+
```
Comment (mporam) [inline edit/about.php]: no DOCTYPE

File: edit/about.php [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+                <label>Name: </label>
+                <input name="name" type="text">
+            </div>
+            <div class="longinput">
+                <label>Title: </label>
+                <input name="title" type="text">
+            </div>
+            <div class="longinput">
+                <label>Bio: </label>
+                <input name="desc" type="">
```
Comment (mporam) [inline edit/about.php]: missing type value

File: edit/addproj.php [hunk 4]
```diff
@@ -0,0 +1,55 @@
+<?php
+/**
```
Comment (mporam) [inline edit/addproj.php]: remove this comment

File: edit/addproj.php [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+ * Created by PhpStorm.
+ * User: academy
+ * Date: 01/10/2018
+ * Time: 10:21
+ */
+
+?>
+
```
Comment (mporam) [inline edit/addproj.php]: missing DOCTYPE

File: edit/addproj.php [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+<html>
+<head>
+    <title>Edit Badges</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+    <body>
+        <form>
+            <ul>
+                <li>
```
Comment (mporam) [inline edit/addproj.php]: do you need the bullet point list? shouldnt do

File: edit/editproj.php [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+?>
+<html>
+<head>
+    <title>Edit/Delete Projects</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <form>
+        <ul>
+            <li>
```
Comment (mporam) [inline edit/editproj.php]: again, you shouldnt need the list

**Review by mporam — CHANGES_REQUESTED** [review]
most comments are applicable to all page builds

File: edit/addproj.php (hunk 5 above)
Comment (hoptond) [reply edit/addproj.php]: resolved in a3903c7

File: edit/about.php (hunk 2 above)
Comment (hoptond) [reply edit/about.php]: resolved in a3903c7

File: edit/about.php (hunk 1 above)
Comment (hoptond) [reply edit/about.php]: resolved in a3903c7

File: edit/addproj.php (hunk 4 above)
Comment (hoptond) [reply edit/addproj.php]: resolved in a3903c7

File: edit/about.php (hunk 3 above)
Comment (hoptond) [reply edit/about.php]: fixed in 5417a87

File: edit/addproj.php (hunk 6 above)
Comment (hoptond) [reply edit/addproj.php]: fixed in 90c0c0f

File: edit/editproj.php (hunk 7 above)
Comment (hoptond) [reply edit/editproj.php]: fixed in 90c0c0f

**Comment by hoptond** [comment]
Fixed the problems you spotted and fixed some other things I hadn't spotted the first time round

**Review by mporam — APPROVED** [review]

**Pull request merged**